2024-01-29 17:49:34,147 INFO    StreamThr :62196 [internal.py:wandb_internal():86] W&B internal server running at pid: 62196, started at: 2024-01-29 17:49:34.145351
2024-01-29 17:49:34,149 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status
2024-01-29 17:49:34,150 INFO    WriterThread:62196 [datastore.py:open_for_write():85] open: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/run-yx1tzgs8.wandb
2024-01-29 17:49:34,152 DEBUG   SenderThread:62196 [sender.py:send():382] send: header
2024-01-29 17:49:34,152 DEBUG   SenderThread:62196 [sender.py:send():382] send: run
2024-01-29 17:49:34,794 INFO    SenderThread:62196 [dir_watcher.py:__init__():211] watching files in: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/files
2024-01-29 17:49:34,795 INFO    SenderThread:62196 [sender.py:_start_run_threads():1136] run started: yx1tzgs8 with start time 1706510973.328289
2024-01-29 17:49:34,810 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: check_version
2024-01-29 17:49:34,810 DEBUG   SenderThread:62196 [sender.py:send_request():409] send_request: check_version
2024-01-29 17:49:35,026 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: run_start
2024-01-29 17:49:35,029 DEBUG   HandlerThread:62196 [system_info.py:__init__():32] System info init
2024-01-29 17:49:35,030 DEBUG   HandlerThread:62196 [system_info.py:__init__():47] System info init done
2024-01-29 17:49:35,030 INFO    HandlerThread:62196 [system_monitor.py:start():194] Starting system monitor
2024-01-29 17:49:35,030 INFO    SystemMonitor:62196 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-01-29 17:49:35,030 INFO    HandlerThread:62196 [system_monitor.py:probe():214] Collecting system info
2024-01-29 17:49:35,031 INFO    SystemMonitor:62196 [interfaces.py:start():190] Started cpu monitoring
2024-01-29 17:49:35,031 DEBUG   HandlerThread:62196 [system_info.py:probe():196] Probing system
2024-01-29 17:49:35,033 INFO    SystemMonitor:62196 [interfaces.py:start():190] Started disk monitoring
2024-01-29 17:49:35,034 INFO    SystemMonitor:62196 [interfaces.py:start():190] Started gpuapple monitoring
2024-01-29 17:49:35,034 INFO    SystemMonitor:62196 [interfaces.py:start():190] Started memory monitoring
2024-01-29 17:49:35,039 INFO    SystemMonitor:62196 [interfaces.py:start():190] Started network monitoring
2024-01-29 17:49:35,042 DEBUG   HandlerThread:62196 [system_info.py:_probe_git():181] Probing git
2024-01-29 17:49:35,079 DEBUG   HandlerThread:62196 [system_info.py:_probe_git():189] Probing git done
2024-01-29 17:49:35,079 DEBUG   HandlerThread:62196 [system_info.py:probe():244] Probing system done
2024-01-29 17:49:35,079 DEBUG   HandlerThread:62196 [system_monitor.py:probe():223] {'os': 'macOS-13.5.1-arm64-arm-64bit', 'python': '3.11.5', 'heartbeatAt': '2024-01-29T06:49:35.033771', 'startedAt': '2024-01-29T06:49:24.742761', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'Documents/GitHub/Data_Distill_with_NC/Data_Distillation_and_Nerual_Collapse_Demo.ipynb', 'codePathLocal': None, 'root': '/Users/songzeyang/Documents/GitHub/Data_Distill_with_NC', 'git': {'remote': 'https://github.com/SongzeYang0721/Data_Distillation_and_Neural_Collpase.git', 'commit': 'bf669acd92582524a2e6484945bdfc4a266936dd'}, 'email': 'yangsongze2008@gmail.com', 'host': 'Songzes-MacBook-Pro.local', 'username': 'songzeyang', 'executable': '/Users/songzeyang/anaconda3/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 460.4317207336426, 'used': 118.50743103027344}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 8.0}}
2024-01-29 17:49:35,079 INFO    HandlerThread:62196 [system_monitor.py:probe():224] Finished collecting system info
2024-01-29 17:49:35,079 INFO    HandlerThread:62196 [system_monitor.py:probe():227] Publishing system info
2024-01-29 17:49:35,079 DEBUG   HandlerThread:62196 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-01-29 17:49:35,079 DEBUG   HandlerThread:62196 [system_info.py:_save_pip():68] Saving pip packages done
2024-01-29 17:49:35,080 DEBUG   HandlerThread:62196 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-01-29 17:49:35,800 INFO    Thread-26 :62196 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/files/requirements.txt
2024-01-29 17:49:35,800 INFO    Thread-26 :62196 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/files/conda-environment.yaml
2024-01-29 17:49:39,269 DEBUG   HandlerThread:62196 [system_info.py:_save_conda():87] Saving conda packages done
2024-01-29 17:49:39,272 INFO    HandlerThread:62196 [system_monitor.py:probe():229] Finished publishing system info
2024-01-29 17:49:39,272 DEBUG   SenderThread:62196 [sender.py:send():382] send: files
2024-01-29 17:49:39,272 INFO    SenderThread:62196 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-01-29 17:49:39,273 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:49:39,284 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: stop_status
2024-01-29 17:49:39,284 DEBUG   SenderThread:62196 [sender.py:send_request():409] send_request: stop_status
2024-01-29 17:49:39,285 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:49:39,560 DEBUG   SenderThread:62196 [sender.py:send():382] send: telemetry
2024-01-29 17:49:39,818 INFO    Thread-26 :62196 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/files/conda-environment.yaml
2024-01-29 17:49:39,819 INFO    Thread-26 :62196 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/files/wandb-metadata.json
2024-01-29 17:49:40,158 INFO    wandb-upload_0:62196 [upload_job.py:push():131] Uploaded file /var/folders/50/9xzt17gx7hz47b3vjql39fjh0000gn/T/tmpp592jci9wandb/mqlkentg-wandb-metadata.json
2024-01-29 17:49:44,581 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:49:49,600 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:49:54,292 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: stop_status
2024-01-29 17:49:54,294 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:49:54,294 DEBUG   SenderThread:62196 [sender.py:send_request():409] send_request: stop_status
2024-01-29 17:49:55,511 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:50:00,529 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:50:05,554 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: status_report
2024-01-29 17:50:05,810 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:05,810 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:07,065 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:07,065 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:09,298 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:50:09,298 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: stop_status
2024-01-29 17:50:09,318 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:09,318 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:09,319 INFO    SenderThread:62196 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql
2024-01-29 17:50:14,315 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:14,380 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:14,380 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:19,335 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:22,948 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:22,948 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:24,298 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:50:29,370 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:34,639 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:35,046 DEBUG   SystemMonitor:62196 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-01-29 17:50:39,303 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:50:41,292 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:50:41,292 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:50:44,674 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:49,690 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:50:54,305 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:51:00,118 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:05,498 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:09,309 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:51:13,819 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:51:13,820 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:51:14,535 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:19,557 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:24,314 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:51:25,436 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: partial_history
2024-01-29 17:51:29,609 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:34,621 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:39,320 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:51:44,654 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:49,674 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:51:54,323 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:51:59,698 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:52:04,720 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:52:09,327 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: internal_messages
2024-01-29 17:52:15,337 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:52:20,364 DEBUG   HandlerThread:62196 [handler.py:handle_request():146] handle_request: keepalive
2024-01-29 17:52:22,229 ERROR   SenderThread:62196 [internal_api.py:execute():373] 409 response executing GraphQL.
2024-01-29 17:52:22,229 ERROR   SenderThread:62196 [internal_api.py:execute():374] {"errors":[{"message":"run data-distillation-with-nc/DD\u0026NC/yx1tzgs8 was previously created and deleted; try a new run name","path":["upsertBucket"]}],"data":{"upsertBucket":null}}
2024-01-29 17:52:22,234 INFO    HandlerThread:62196 [handler.py:finish():866] shutting down handler
2024-01-29 17:52:22,235 INFO    HandlerThread:62196 [system_monitor.py:finish():203] Stopping system monitor
2024-01-29 17:52:22,235 INFO    WriterThread:62196 [datastore.py:close():294] close: /Users/songzeyang/Documents/GitHub/Data_Distill_with_NC/wandb/run-20240129_174924-yx1tzgs8/run-yx1tzgs8.wandb
2024-01-29 17:52:22,237 DEBUG   SystemMonitor:62196 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-01-29 17:52:22,237 INFO    HandlerThread:62196 [interfaces.py:finish():202] Joined cpu monitor
2024-01-29 17:52:22,237 DEBUG   SystemMonitor:62196 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-01-29 17:52:22,238 INFO    HandlerThread:62196 [interfaces.py:finish():202] Joined disk monitor
2024-01-29 17:52:22,239 INFO    HandlerThread:62196 [interfaces.py:finish():202] Joined gpuapple monitor
2024-01-29 17:52:22,240 INFO    HandlerThread:62196 [interfaces.py:finish():202] Joined memory monitor
2024-01-29 17:52:22,240 INFO    HandlerThread:62196 [interfaces.py:finish():202] Joined network monitor
2024-01-29 17:52:22,802 ERROR   StreamThr :62196 [internal.py:wandb_internal():174] Thread SenderThread:
Traceback (most recent call last):
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/apis/normalize.py", line 41, in wrapper
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 2212, in upsert_run
    response = self.gql(
               ^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 341, in gql
    ret = self._retry_gql(
          ^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_api.py", line 369, in execute
    return self.client.execute(*args, **kwargs)  # type: ignore
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 52, in execute
    result = self._get_result(document, *args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/vendor/gql-0.2.0/wandb_gql/client.py", line 60, in _get_result
    return self.transport.execute(document, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/lib/gql_request.py", line 59, in execute
    request.raise_for_status()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 409 Client Error: Conflict for url: https://api.wandb.ai/graphql

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_util.py", line 49, in run
    self._run()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal_util.py", line 93, in _run
    self._debounce()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/internal.py", line 334, in _debounce
    self._sm.debounce()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/sender.py", line 540, in debounce
    self._maybe_update_config(always=final)
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/sender.py", line 517, in _maybe_update_config
    self._debounce_config()
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/sender.py", line 546, in _debounce_config
    self._api.upsert_run(
  File "/Users/songzeyang/anaconda3/lib/python3.11/site-packages/wandb/apis/normalize.py", line 51, in wrapper
    raise CommError(message, error)
wandb.errors.CommError: run data-distillation-with-nc/DD&NC/yx1tzgs8 was previously created and deleted; try a new run name (Error 409: Conflict)
